tasks/main.yml

```diff
@@ -23,6 +33,9 @@
     version: "{{ item.version | default('') }}"
     global: yes
     state: present
+  become: yes
+  become_user: "{{ nodejs_install_npm_user }}"
+  when: nodejs_forever
```

Comment:
This conditional shouldn't be included; global package installs defined in the playbook should be installed regardless of whether `forever` is installed separately.